From 48726b86cbd385e7722cd1d7c342097058b78157 Mon Sep 17 00:00:00 2001
From: chpr <carthango@gmx.net>
Date: Sun, 10 Jan 2021 22:45:49 +0100
Subject: [PATCH 4/5] Check for sys/param on configure

Prevent compiler warning and check for sys/param on configure
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 8278353d4..daf857bea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -442,6 +442,7 @@ AC_CHECK_HEADERS([linux/input.h], [HAVE_EVDEV=1], [HAVE_EVDEV=0])
 AM_CONDITIONAL([HAVE_EVDEV], [test "x$HAVE_EVDEV" = "x1"])
 
 AC_CHECK_HEADERS_ONCE([sys/prctl.h])
+AC_CHECK_HEADERS_ONCE([sys/param.h])
 
 # Solaris
 AC_CHECK_HEADERS_ONCE([sys/conf.h sys/filio.h])
-- 
2.29.2.windows.2

